packages/salesforcedx-vscode-core/test/commands/isvdebugger/bootstrapCmd.test.ts

```diff
@@ -117,6 +117,30 @@ describe('ISV Debugging Project Bootstrap Command', () => {
         expect.fail('Response should be of type ContinueResponse');
       }
     });
+
+    it('Should accept valid URI', async () => {
+      const response = EnterForceIdeUri.uriValidator(
+        `forceide://abc?url=${LOGIN_URL}&sessionId=${SESSION_ID}`
+      );
+      expect(response).to.be.null;
+    });
+
+    it('Should complain about invalid URI', async () => {
+      expect(
+        EnterForceIdeUri.uriValidator(
+          `forceide://abc?url=${LOGIN_URL}&missingSessionId`
+        )
+      ).to.not.be.null;
```

Comment:
Do these return `nls.localize('parameter_gatherer_invalid_forceide_url')`?

Reply:
Yes. Do you prefer checking agains the full message? I was just verifying that they return something - which VS Code does imply as error

Reply:
Verifying the message sounds good.